src/clang-fetcher.ts

```diff
@@ -102,20 +102,20 @@ async function downloadClangVersion(electronVersion: string) {
   d('fetching clang:', clangVersionString);
 
   const clangDownloadURL = getClangDownloadURL('clang', clangVersionString, process.platform, process.arch);
-  
+
   const contents = await fetch(clangDownloadURL, 'buffer');
   d('deflating clang');
   zlib.deflateSync(contents);
   const tarPath = path.resolve(ELECTRON_GYP_DIR, `${electronVersion}-clang.tar`);
-  if (await fs.pathExists(tarPath)) await fs.remove(tarPath);
-  await fs.writeFile(tarPath, Buffer.from(contents));
-  await fs.mkdirp(clangDirPath);
+  if (fs.existsSync(tarPath)) await fs.promises.rm(tarPath, { recursive: true, force: true });
+  await fs.promises.writeFile(tarPath, Buffer.from(contents));
```

Comment:
Since this is a graceful-fs call, it'll need to be passed through util.promisify